examples/gno.land/p/demo/airdrop/merkle-airdrop.gno

```diff
@@ -0,0 +1,88 @@
+package airdrop
```

Comment:
`p/demo/airdrop` seems to be too generic IMO, so I would either name it accurately like `p/demo/grc20_merkle_airdrop` or use a subpaths, maybe `p/demo/airdrop/grc20_merkle`.